src/support/bytemuck.rs

```diff
@@ -0,0 +1,66 @@
+//! Support for the [`bytemuck`](https://crates.io/crates/bytemuck) crate.
+#![cfg(feature = "bytemuck")]
+#![cfg_attr(docsrs, doc(cfg(feature = "bytemuck")))]
+
+use bytemuck::Pod;
+use ruint::Uint;
+
+// Implement the `Pod` trait for `Uint` types with a size that is a multiple of 64, up to 1024.
+macro_rules! impl_pod {
+    ($($bits:expr),+ $(,)?) => {
+        $(
+            unsafe impl<const LIMBS: usize> bytemuck::Zeroable for Uint<{$bits}, LIMBS> {}
```

Comment:
`Zeroable` should be defined once generically for all Uint. It does not require the BITS to be a multiple of `64` so should not be part of this macro.